src/backend/aspen/app/views/tests/test_samples_view.py

```diff
@@ -25,7 +25,13 @@ def test_samples_view(
     group = group_factory()
     user = user_factory(group)
     sample = sample_factory(group)
-    uploaded_pathogen_genome = uploaded_pathogen_genome_factory(sample)
+    pathogen_genome = uploaded_pathogen_genome_factory(
```

Comment:
i don't agree with the rename of `uploaded_pathogen_genome` -> `pathogen_genome`.

Reply:
changed it back